analytics.js

```diff
@@ -182,11 +183,14 @@ class Analytics {
       );
 
       // remove from the list which don't have support yet in SDK
-      this.clientIntegrations = this.clientIntegrations.filter(intg => {
+      this.clientIntegrations = this.clientIntegrations.filter((intg) => {
         return integrations[intg.name] != undefined;
       });
-
-      this.init(this.clientIntegrations);
+      const enabledIntegrations = getIntegrationsAfterCookieConsent(
+        this.clientIntegrations,
+        response
+      );
+      this.init(enabledIntegrations);
```

Comment:
Let's not do this as `this.clientIntegrations` is used elsewhere too but rather do it prior to Ln 180 (take response and `loadOnlyIntegrations` as input)